cli/src/commands/content-directory/createEntity.ts

```diff
@@ -0,0 +1,170 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { flags } from '@oclif/command'
+import { Class as ContentDirectoryClass } from '@joystream/types/content-directory'
+import inquirer from 'inquirer'
+import { CLIError } from '@oclif/errors'
+import { InputParser } from '@joystream/cd-schemas'
+
+const CONTEXTS = ['Member', 'Curator', 'Lead'] as const
+
+type CreateEntityArgs = {
+  className: string
+}
+
+type CreateEntityFlags = {
+  context: typeof CONTEXTS[number]
+}
+
+type Question = {
```

Comment:
Can we instead use a type that comes directly from `inquirer` library like `DistinctQuestion`, ie.:
https://github.com/Joystream/joystream/blob/1ee71fe41117d317fa8f187fffe1df124f0737d9/cli/src/base/DefaultCommandBase.ts#L3

I suspect this type may cause some issues if we just use it as `DistinctQuestion[]`, since `inquirer` uses quite complex generic typings and I ran into some issues with that in the past too, but if we could use some type that comes directly from the library - that would be the best approach.

Otherwise we should only define a type like this in one place (ie. `cli/src/Types.ts`)